pages/opportunities/OpportunitiesPage.jsx

```diff
@@ -29,8 +29,8 @@ var OpportunitiesPage = React.createClass({
                 imgSrc="/img/pages/opportunities/svg/icon-leadership-twitter.svg"
                 width={60}
                 head="Follow Us"
-                subhead="We're @mozteach on Twitter and our community uses #teachtheweb"
-                highlightedText="@mozteach"
+                subhead="We're @mozlearn on Twitter and our community uses #teachtheweb"
```

Comment:
same note as above - we can pull in the `config` and then get the twitter handle from that so that if we need to ever change it again, this file won't need another update